### ISSUE TITLE
HDF5 - ld problem (build failure)

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/hdf5-deploy#3]
Builds failing with 

```
  FCLD     libhdf5_fortran.la

/usr/bin/ld: cannot find -l-L/apprepo/generic/u1404/x86_64/gcc/4.9.2/lib/gcc/x86_64-unknown-linux-gnu/4.9.2
collect2: error: ld returned 1 exit status
```

Looks like the modulefile either for gcc or openmpi is borked.

**Comment by brucellino** [comment]
Looks like this problem is reported here : http://lists.hdfgroup.org/pipermail/hdf-forum_lists.hdfgroup.org/2014-February/007598.html

**Comment by brucellino** [comment]
fixed in  8a29ac5
